service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/OutsourcedCaseSentToGatekeepingContentProviderTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+import uk.gov.hmcts.reform.ccd.model.OrganisationPolicy;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Hearing;
+import uk.gov.hmcts.reform.fpl.model.Orders;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.notify.sendtogatekeeper.NotifyLAOnOutsourcedCaseTemplate;
+import uk.gov.hmcts.reform.fpl.utils.ElementUtils;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.OrderType.CARE_ORDER;
+import static uk.gov.hmcts.reform.fpl.enums.OrderType.EMERGENCY_PROTECTION_ORDER;
+import static uk.gov.hmcts.reform.fpl.enums.YesNo.NO;
+import static uk.gov.hmcts.reform.fpl.enums.YesNo.YES;
+
+@ContextConfiguration(classes = {OutsourcedCaseSentToGatekeepingContentProvider.class})
+class OutsourcedCaseSentToGatekeepingContentProviderTest extends AbstractEmailContentProviderTest {
+    private static final String THIRD_PARTY_ORG_NAME = "External org";
+    private static final Long CASE_ID = 12345L;
+
+    @Autowired
+    OutsourcedCaseSentToGatekeepingContentProvider outsourcedCaseSentToGatekeepingContentProvider;
+
+    @Test
+    void shouldBuildNotifyLAOnOutsourcedCaseTemplateWithOutsourcedOrganisation() {
+        CaseData caseData = CaseData.builder()
+            .id(CASE_ID)
+            .respondents1(List.of(ElementUtils.element(Respondent.builder()
+                .party(RespondentParty.builder()
+                    .firstName("Mark")
+                    .lastName("Watson")
+                    .build())
+                .build())))
+            .hearing(Hearing.builder()
+                .timeFrame("Same day")
+                .build())
+            .orders(Orders.builder()
+                .orderType(List.of(CARE_ORDER, EMERGENCY_PROTECTION_ORDER))
+                .build())
+            .outsourcingPolicy(OrganisationPolicy.builder()
+                .organisation(Organisation.builder()
+                    .organisationName(THIRD_PARTY_ORG_NAME)
+                    .build())
+                .build())
+            .build();
+
+        NotifyLAOnOutsourcedCaseTemplate actualTemplate = outsourcedCaseSentToGatekeepingContentProvider
+            .buildNotifyLAOnOutsourcedCaseTemplate(caseData);
+
+        assertThat(actualTemplate).usingRecursiveComparison().isEqualTo(getExpectedTemplate());
```

Comment:
usingRecursiveComparision is not needed

it would be better to add
@ToString(callSuper = true)
to NotifyLAOnOutsourcedCaseTemplate